Clean Swift/Models.xctemplate/___FILEBASENAME___Models.swift

```diff
@@ -8,11 +8,18 @@
 
 import UIKit
 
-struct ___FILEBASENAMEASIDENTIFIER___Request {
-}
-
-struct ___FILEBASENAMEASIDENTIFIER___Response {
-}
-
-struct ___FILEBASENAMEASIDENTIFIER___ViewModel {
+struct ___FILEBASENAMEASIDENTIFIER___
+{
+    struct Something
```

Comment:
I think it's nested too much. You don't usually use multiple requests in one scene.